modules/options/src/main/scala/scala/build/Positioned.scala

```diff
@@ -20,6 +20,8 @@ final case class Positioned[+T](
       pos0.value
     )
   }
+
+  // override def toString = s"$value at positions ${positions.mkString(", ")}"
```

Comment:
```suggestion
```

Reply:
I need to make sure, that the commenting is not for debugging purposes.